### ISSUE TITLE
landscape: constant polling

### ISSUE THREAD

**Issue by tylershuster** [urbit/urbit#3711]
This happens all the time in landscape. I don't think it should be.

![Screen Recording 2020-10-14 at 10 02 09 AM](https://user-images.githubusercontent.com/3999320/96021720-b092c980-0e04-11eb-9b82-80d6ec0785b2.gif)

**Comment by tacryt-socryp** [comment]
Server-side events not working well in HTTP 1.1 is a known issue. A good question, though, is what to do about it (either by changing our messaging, or otherwise)

**Comment by joemfb** [comment]
Those are PUTs, so isn't this eager acknowledgement as per #3671?

**Comment by tylershuster** [comment]
Here's the stack trace for one:
```
	sendJSONToChannel
http://localhost/~landscape/js/channel.js:175:11
connectIfDisconnected/this.eventSource.onmessage
http://localhost/~landscape/js/channel.js:221:14

```

I did have problems with landscape constantly rerendering before, but I don't think there was outgoing stuff then.

**Comment by matildepark** [comment]
Is this when disconnected or all the time? Eager acks shouldn't be just nonstop polling, no?

**Comment by tylershuster** [comment]
All day every day

**Comment by matildepark** [comment]
I don't see anything when I record in Safari, no network requests or anything.

**Comment by tylershuster** [comment]
I think this is related to https://github.com/urbit/urbit/issues/3621 and I will investigate accordingly

**Comment by tylershuster** [comment]
Here's the event that keeps getting received over the `eventSource`:

```
diff 
message
​
bubbles: false
​
cancelBubble: false
​
cancelable: false
​
composed: false
​
currentTarget: null
​
data: "{\"json\":{\"groupUpdate\":{\"initialGroup\":{\"resource\":{\"name\":\"poetry\",\"ship\":\"marpem\"},\"group\":{\"tags\":{\"admin\":[\"marpem\",\"radbur-sivmus\"]},\"policy\":{\"open\":{\"banned\":[],\"banRanks\":[]}},\"members\":[\"marpem\",\"radbur-sivmus\"],\"hidden\":true}}}},\"id\":2,\"response\":\"diff\"}"
​
defaultPrevented: false
​
eventPhase: 0
​
explicitOriginalTarget: EventSource { url: "http://localhost/~/channel/1602715384170-c8c738", withCredentials: true, readyState: 1, … }
​
isTrusted: true
​
lastEventId: "28"
​
origin: "http://localhost"
​
originalTarget: EventSource { url: "http://localhost/~/channel/1602715384170-c8c738", withCredentials: true, readyState: 1, … }
​
ports: Array []
​
returnValue: true
​
source: null
​
srcElement: EventSource { url: "http://localhost/~/channel/1602715384170-c8c738", withCredentials: true, readyState: 1, … }
​
target: EventSource { url: "http://localhost/~/channel/1602715384170-c8c738", withCredentials: true, readyState: 1, … }
​
timeStamp: 13080
​
type: "message"
​
<get isTrusted()>: function isTrusted()
​
<prototype>: MessageEventPrototype { initMessageEvent: initMessageEvent(), data: Getter, origin: Getter, … }
channel.js:220:17

```

**Comment by matildepark** [comment]
Oh is this another one of your busted groups?

— 
~haddef-sigwen
https://urbit.org

**Comment by tylershuster** [comment]
Seems like it. Except I'm still getting it after deleting the group. how to leave

**Comment by tylershuster** [comment]
This appears to be fixed in the latest OTA...would love any insight why, closing issue regardless

**Comment by tylershuster** [comment]
But I haven't even received said OTA...?

**Comment by tylershuster** [comment]
red herring

**Comment by tylershuster** [comment]
Fixed busted group (see #3716) now this doesn't happen.
